test/accountSpec.js

```diff
@@ -81,7 +82,15 @@ describe('Accounts', function () {
       yield this.request()
         .get('/accounts')
         .auth('admin', 'admin')
-        .expect([account1, account2, account3, account4, account5, account6])
+        .expect((res) => {
+          const sortedResponse = _.sortBy(res.body, (account) => {
```

Comment:
Insertion order is no longer guaranteed, so we need to sort before we compare the arrays.